### PR TITLE
feat: remove dependency on github artifacts

### PR DESCRIPTION
**Pull request by loispostula**
needed for https://github.com/ForesightMiningSoftwareCorporation/action-trigger-signer-workflow/pull/18
and https://github.com/orica-digital/fslabs-binary_signer/pull/18/files

### CODE REVIEW

File: .github/workflows/rust_binary_publish.yml [hunk 1]
```diff
@@ -155,6 +156,22 @@ jobs:
           sas=$(az storage blob generate-sas --connection-string "${{ secrets.ARTIFACTS_CONNECTION_STRING }}" --container-name ${{ vars.ARTIFACTS_CONTAINER }} --name ${{  inputs.package }}/${{ inputs.release_channel }}/${{  inputs.package }}-${{ matrix.target }}-${{  inputs.toolchain}}-v${{  inputs.version }}${{ fromJson(inputs.targets_config)[matrix.target].extension }} --permissions r --expiry ${{ steps.sas-expiry.outputs.expiry }} --https-only --full-uri --output tsv)
           echo "sas=$sas" >> $GITHUB_OUTPUT
 
+      - name: Generate SAS Url for binary signing upload
+        if: inputs.sign_build == 'true'
+        shell: bash
+        id: sas-upload-url
+        run: |
+          sas=$(az storage blob generate-sas --connection-string "${{ secrets.ARTIFACTS_CONNECTION_STRING }}" --container-name ${{ vars.ARTIFACTS_CONTAINER }} --name ${{  inputs.package }}/${{ inputs.release_channel }}/${{  inputs.package }}-${{ matrix.target }}-${{ inputs.toolchain }}-v${{ inputs.version }}-signed${{ fromJson(inputs.targets_config)[matrix.target].extension }} --permissions w --expiry ${{ steps.sas-expiry.outputs.expiry }} --https-only --full-uri --output tsv)
```
Comment (nathan-tarr-orica) [inline .github/workflows/rust_binary_publish.yml:164]: Might want to put double quotes around the blob name (--name) if it can contain spaces

**Review by nathan-tarr-orica — APPROVED** [review]

File: .github/workflows/rust_binary_publish.yml (hunk 1 above)
Comment (nathan-tarr-orica) [inline .github/workflows/rust_binary_publish.yml:164]:
```suggestion
          sas=$(az storage blob generate-sas --connection-string "${{ secrets.ARTIFACTS_CONNECTION_STRING }}" --container-name ${{ vars.ARTIFACTS_CONTAINER }} --name "${{  inputs.package }}/${{ inputs.release_channel }}/${{  inputs.package }}-${{ matrix.target }}-${{ inputs.toolchain }}-v${{ inputs.version }}-signed${{ fromJson(inputs.targets_config)[matrix.target].extension }}" --permissions w --expiry ${{ steps.sas-expiry.outputs.expiry }} --https-only --full-uri --output tsv)
```

File: .github/workflows/rust_binary_publish.yml [hunk 2]
```diff
@@ -155,6 +156,22 @@ jobs:
...
+          echo "sas=$sas" >> $GITHUB_OUTPUT
+
+      - name: Generate SAS Url for checksum binary signing upload
+        if: inputs.sign_build == 'true'
+        shell: bash
+        id: sas-sha-upload-url
+        run: |
+          sas=$(az storage blob generate-sas --connection-string "${{ secrets.ARTIFACTS_CONNECTION_STRING }}" --container-name ${{ vars.ARTIFACTS_CONTAINER }} --name ${{  inputs.package }}/${{ inputs.release_channel }}/${{  inputs.package }}-${{ matrix.target }}-${{ inputs.toolchain }}-v${{ inputs.version }}-signed-sha512.txt --permissions w --expiry ${{ steps.sas-expiry.outputs.expiry }} --https-only --full-uri --output tsv)
```
Comment (nathan-tarr-orica) [inline .github/workflows/rust_binary_publish.yml:172]:
```suggestion
          sas=$(az storage blob generate-sas --connection-string "${{ secrets.ARTIFACTS_CONNECTION_STRING }}" --container-name ${{ vars.ARTIFACTS_CONTAINER }} --name "${{  inputs.package }}/${{ inputs.release_channel }}/${{  inputs.package }}-${{ matrix.target }}-${{ inputs.toolchain }}-v${{ inputs.version }}-signed-sha512.txt" --permissions w --expiry ${{ steps.sas-expiry.outputs.expiry }} --https-only --full-uri --output tsv)
```

**Comment by loispostula** [comment]
Can be closed, we don't use this anymore

**Pull request closed without merging**